ci/README.md

```diff
@@ -0,0 +1,33 @@
+# Stargate Continuous Integration
+
+This folder contains necessary files for the docker image being used during the CI:
+
+* `Dockerfile` - docker file for the container that will be used during the CI
+* `test.sh` - script that is actually executed during the CI
+* `Makefile` - small Makefile that enables easier building and pushing of a new CI docker image
+
+*Note that we run the CI on the GCP using [Cloud Build](https://cloud.google.com/build).
+The docker image is directly referenced in the [cloudbuild.yaml](../cloudbuild.yaml), [cloudbuild-3_11.yaml](../cloudbuild-3_11.yaml), , [cloudbuild-4_0.yaml](../cloudbuild-4_0.yaml) and [cloudbuild-dse.yaml](../cloudbuild-dse.yaml) files which represent the jobs we do during the CI.*
+
+## Updating the docker image
+
+### Prerequisites
+
+In order to push the new docker image to the GCP image registry `gcr.io/stargateio`, besides having permissions to do so, you should authenticate and setup your docker configuration using:
+
+```bash
+gcloud auth login
+gcloud auth configure-docker
+```
+
+### Process
+
+* Perform wanted updates related to the image.
+* Bump the `VERSION` variable in the [Makefile](Makefile).
+**This step is mandatory, otherwise you will overwrite the existing image.**
```

Comment:
Why not make it a bash script that takes a version as input? I guess for the super paranoid you could always query GCR first and make sure the version doesn't exist before you tag and push.

Reply:
I wanted to avoid the bash script, because with the Makefile it's cleaner imo and you can do `make build` or `make push`, and it's kind of easy :) But yea if you think script would be better I am also fine with this..

I can query GCR first, this is also very reasonable imo..

Reply:
Done with the question and option to abort:

```
[10:08:35] ise:ci <ise-1520-ci-image-update>$ make push
The image with the version v1.0.6 exists, do you want to overwrite [Y/n]?
```